### PR TITLE
Clean up both the default and extended static metadata

### PR DESCRIPTION
**Pull request by igarousi**
List of changes:

- Reduced the significant figures by adding the `significant_figures` property to the `swotVariables` and `swordVariables` in `hydrologic.js` to make the number of significant figures configurable for each variable. 
- Updated the `getSwordDescriptions` function in the `hydrologic.js` to format numerical values to the desired number of significant figures.
- Added `swotviz_alias` property to the `swotVariables` and `swordVariables` in `hydrologic.js` to display a shorter key names for metadata. 
- Updated the `getSwordDescriptions` function in `hydrologic.js` to use the alias property.
- Updated the template in `StaticMetadata.vue` to display alias property instead of short_definition.
- Added a `mapping` property to the `swotVariables` and `swordVariables` in `hydrologic.js` to display the real names rather than code numbers (e.g., Reach Type = River instead of Reach Typle = 1).
- Updated the `getSwordDescriptions` function in `hydrologic.js` to use the mapping property.
- Added a small "info" icon (using mdi-information-outline from Material Design Icons) to the `StaticMetadata.vue` to show a full description next to each metadata variable.

### CODE REVIEW

**Comment by igarousi** [comment]
@devincowan could you please review these changes?

File: frontend/src/stores/hydrologic.js [hunk 1]
```diff
@@ -417,19 +493,23 @@ export const useHydrologicStore = defineStore('hydrologic', () => {
...
         'The maximum number of SWOT passes to intersect each reach during the 21 day orbit cycle',
+      swotviz_alias: 'SWOT Pass Count',
       units: '',
-      plottable: false
+      plottable: false,
+      significant_figures: 0 
     },
     {
-      abbreviation: 'swot_orbits',
+      abbreviation: 'swot_orbit',
```
Comment (devincowan) [inline frontend/src/stores/hydrologic.js:502]: image
do we want this to be delimited in any way? (just for display)

**Review by devincowan — APPROVED** [review]
This is awesome, thanks @igarousi

File: frontend/src/stores/hydrologic.js [hunk 2]
```diff
@@ -271,27 +292,33 @@ export const useHydrologicStore = defineStore('hydrologic', () => {
...
       abbreviation: 'n_chan_mod',
       definition: 'Mode of the number of channels for each node or reach',
       fileType: 'all',
       default: false,
       short_definition: 'Mode of the number of channels for each node or reach',
+      swotviz_alias: 'Mode Channel Count',
       units: '',
-      plottable: false
+      plottable: false,
+      significant_figures: 0 //TODO: Check this
```
Comment (devincowan) [inline frontend/src/stores/hydrologic.js:321]: you have a bunch of `TODO`s here...
the sig figs look good to me.
maybe remove the Todos if no longer applicable?

File: frontend/src/stores/hydrologic.js (hunk 1 above)
Comment (igarousi) [reply frontend/src/stores/hydrologic.js:502]: That's a great idea!

File: frontend/src/stores/hydrologic.js (hunk 2 above)
Comment (igarousi) [reply frontend/src/stores/hydrologic.js]: Removed TODOs.

File: frontend/src/stores/hydrologic.js (hunk 1 above)
Comment (igarousi) [reply frontend/src/stores/hydrologic.js:502]: Done!

**Comment by devincowan** [comment]
looks great, merge at your leisure @igarousi!

**Pull request merged**